styles.css

```diff
@@ -1,4 +1,4 @@
-@media (min-width: 375px) and (max-width: 768px) {
+@media (min-width: 375px) {
   body {
     margin: 0;
   }
```

Comment:
- [ ] Kindly, there's an issue with the mobile version of your portfolio, it's not displayed correctly. I recommend checking the breakpoint.

Reply:
👉 Mr. Ricardo could you change the breakpoint from `min-width: 375px` to `max-width: 375px`  and then try to see the difference I hope it may correct the layout.